src/main/java/org/opentripplanner/middleware/models/MonitoredTrip.java

```diff
@@ -279,5 +307,61 @@ public boolean delete() {
         // TODO: Add journey state deletion.
         return Persistence.monitoredTrips.removeById(this.id);
     }
+
+    public Map<String, String> parseQueryParams() throws URISyntaxException {
+        return URLEncodedUtils.parse(
+            new URI(String.format("http://example.com/plan?%s", queryParams)),
+            UTF_8
+        ).stream().collect(Collectors.toMap(NameValuePair::getName, NameValuePair::getValue));
+    }
+
+    public boolean isArriveBy() throws URISyntaxException {
+        // if arriveBy is not included in query params, OTP will default to false, so initialize to false
+        return parseQueryParams().getOrDefault("arriveBy", "false").equals("true");
+    }
+
+    /**
+     * Gets the timezone of the target location. When planning a trip, we either want to depart at a certain time or 
+     * arrive by a certain time. When departing, we use the local time present at the origin and when arriving by we 
+     * use the local time at the destination. Therefore, this method will return the timezone at the destination if this
+     * trip is an arriveBy trip, or the timezone at the origin if the trip is a depart at trip.
```

Comment:
As long as this is how OpenTripPlanner handles this, I'm OK with this approach. Can you confirm with a code ref? (Might be good to include in this javadoc as well.)

Reply:
Le sigh, OTP does not handle it this way... See https://github.com/ibi-group/OpenTripPlanner/blob/8ec56c10702413f19bcea32e40d1ede0a18ee3f0/src/main/java/org/opentripplanner/routing/graph/Graph.java#L955-L984. The Itinerary responses don't return a timezone identifier, so I think we're going to have to set one in the config.

Reply:
Refactored in 02379c18665691badae829898367975031e3ce37